Create ui-c8y-1021-77-2-replace-single-operations-list-with-data-grid-in-bulk-operations-overview.md (#3408)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-77-2-replace-single-operations-list-with-data-grid-in-bulk-operations-overview.md

---
date: ""

> title: List reload loop in bulk operation details view has been fixed

Reviewer: ```suggestion
title: Improved operations list reload loop in bulk operations details
```

> title: List reload loop in bulk operation details view has been fixed

Reviewer 2: ```suggestion
title: Improved operations list reload functionality in bulk operations details
```

> title: Replace single operations list with data grid in bulk operations overview

Reviewer 2: ```suggestion
title: List reload loop in bulk operation details view has been fixed
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4720
version: 1021.77.2
---

> Replace single operations list with data grid in bulk operations overview

Reviewer 2: ```suggestion
In the expanded details section of a bulk operation in the **Bulk operations** view, the list of single operations was reloaded too frequently for bulk operations with short delay interval between the single operations which made interaction with the view difficult in the time single operations were created. To avoid this, a throttling of 30 sec. has been introduced. Bulk operation details are still updated in realtime (when activated), the operations list is however is re-loaded only every 30 sec. (or rarely if the delay time is higher). Additionally, the operations list has been replaced by a data grid which makes navigation through large lists of single operations better.
```

> In the expanded details section of a bulk operation in the **Bulk operations** view, the list of single operations was reloaded too frequently for bulk operations with short delay interval between the single operations which made interaction with the view difficult in the time single operations were created. To avoid this, a throttling of 30 sec. has been introduced. Bulk operation details are still updated in realtime (when activated), the operations list is however is re-loaded only every 30 sec. (or rarely if the delay time is higher). Additionally, the operations list has been replaced by a data grid which makes navigation through large lists of single operations better.

Reviewer: ```suggestion
In the details section of a bulk operation in the **Bulk operations** page, the list of single operations was reloaded too frequently for bulk operations with a short delay interval between the single operations. This made interaction with the page difficult when single operations were created. To avoid this, a throttling of 30 seconds has been introduced. Bulk operation details are still updated in real-time (when activated), but the operations list is only reloaded every 30 seconds (or rarer if the delay time is higher). Additionally, the operations list has been replaced with a data grid, which improves navigation through large lists of single operations.
```
